### ISSUE TITLE
CMake over Visual Studio Sollutions

### ISSUE THREAD

**Issue by viloskovic** [oreonengine/OREONENGINE-Cpp#1]
I have done some little project in C++ before, nothing mention-worthy.
But since I was using (again) Linux, I felt the need to not use Visual Studio at all.
Yet the Visual Studio Solutions are only readable by Visual Studio if I am correct.

To support cross-platform and target a larger group of contributors, it can be interesting to change your build tool from Visual Studio to CMake. Visual Studio is still able to read CMake projects and QT-tools can also read CMake projects.

My skills with CMake are not that good so I don't think I can help you with that in the near future.

**Comment by fynnfluegge** [comment]
Yes sure I have some experience with CMake and I want to create a CMake built for the project, but first I want to setup a running prototype engine, after that I will focus on stuff like CMake etc
